gcp_resource_count.sh

```diff
@@ -23,6 +25,7 @@ _make_temp_file() {
 }
 
 PROJECT_ID=""
+MAX_DB_SIZE=1100
```

Comment:
1 TB is 1024 GB.

Reply:
Yea I know but we add a buffer in CloudSQL because of how they calculate disk size